### PR TITLE
Update with-svelte.mdx

### PR DESCRIPTION
**Pull request by shinriyo**
## What kind of change does this PR introduce?

Fix docs.

## What is the current behavior?

double quote
Hard to understand the document.

## What is the new behavior?

removed the double quote.
More understandable document.

## Additional context

It is good for beginner.

### CODE REVIEW

File: apps/docs/pages/guides/getting-started/tutorials/with-svelte.mdx [hunk 1]
```diff
@@ -358,13 +358,17 @@ And then we can add the widget to the Account page:
 
 ```html src/lib/Account.svelte
 <script lang="ts">
+  // Other import elements
+
   // Import the new component
   import Avatar from './Avatar.svelte'
+
+  // Other elements
```
Comment (charislam) [inline apps/docs/pages/guides/getting-started/tutorials/with-svelte.mdx:366]:
```suggestion
  // Import the new component
  import Avatar from './Avatar.svelte'
```

I personally think it's less confusing without this, but not a strong opinion

**Review by charislam — APPROVED** [review]
Thanks for the PR! Agreed it's better without the quotes. Added a typo fix and minor suggestion

File: apps/docs/pages/guides/getting-started/tutorials/with-svelte.mdx [hunk 2]
```diff
@@ -272,9 +272,9 @@ Let's create an avatar for the user so that they can upload a profile photo. We
   import { supabase } from '../supabaseClient'
 
   export let size: number
-  export let url: string
+  export let url: string | null = null
 
-  let avatarUrl: string = null
+  let avatarUrl: string | null = nul
```
[the remark this answers is not in the document]
Comment (shinriyo) [reply apps/docs/pages/guides/getting-started/tutorials/with-svelte.mdx:277]: Thank you.

**Comment by fsansalvadore** [comment]
@shinriyo please run `npm run format` in the root dir to fix the Prettier error, then we can merge this PR.

**Comment by shinriyo** [comment]
I'm sorry. I didn't check the reply.

**Pull request merged**